cri/v1alpha2/cri.go

```diff
@@ -406,57 +430,57 @@ func (c *CriManager) StopPodSandbox(ctx context.Context, r *runtime.StopPodSandb
 
 	containers, err := c.ContainerMgr.List(ctx, opts)
 	if err != nil {
-		return nil, fmt.Errorf("failed to stop sandbox %q: %v", podSandboxID, err)
+		return nil, fmt.Errorf("failed to stop sandbox %q: %v", sandboxContainerID, err)
 	}
 
 	// Stop all containers in the sandbox.
 	for _, container := range containers {
 		err = c.ContainerMgr.Stop(ctx, container.ID, defaultStopTimeout)
 		if err != nil {
-			return nil, fmt.Errorf("failed to stop container %q of sandbox %q: %v", container.ID, podSandboxID, err)
+			return nil, fmt.Errorf("failed to stop container %q of sandbox %q: %v", container.ID, sandboxContainerID, err)
 		}
-		logrus.Infof("success to stop container %q of sandbox %q", container.ID, podSandboxID)
+		logrus.Infof("success to stop container %q of sandbox %q", container.ID, sandboxContainerID)
 	}
 
-	container, err := c.ContainerMgr.Get(ctx, podSandboxID)
+	container, err := c.ContainerMgr.Get(ctx, sandboxContainerID)
 	if err != nil {
 		return nil, err
 	}
 	metadata, err := parseSandboxName(container.Name)
 	if err != nil {
-		return nil, fmt.Errorf("failed to parse metadata of sandbox %q from container name: %v", podSandboxID, err)
+		return nil, fmt.Errorf("failed to parse metadata of sandbox %q from container name: %v", sandboxContainerID, err)
 	}
 
 	securityContext := sandboxMeta.Config.GetLinux().GetSecurityContext()
 	hostNet := securityContext.GetNamespaceOptions().GetNetwork() == runtime.NamespaceMode_NODE
 
 	// Teardown network of the pod, if it is not in host network mode.
 	if !hostNet {
-		sandbox, err := c.ContainerMgr.Get(ctx, podSandboxID)
+		sandbox, err := c.ContainerMgr.Get(ctx, sandboxContainerID)
 		if err != nil {
-			return nil, fmt.Errorf("failed to get sandbox %q: %v", podSandboxID, err)
+			return nil, fmt.Errorf("failed to get sandbox %q: %v", sandboxContainerID, err)
 		}
 
 		netNSPath := containerNetns(sandbox)
 		err = c.CniMgr.TearDownPodNetwork(&ocicni.PodNetwork{
 			Name:         metadata.GetName(),
 			Namespace:    metadata.GetNamespace(),
-			ID:           podSandboxID,
+			ID:           sandboxContainerID,
 			NetNS:        netNSPath,
 			PortMappings: toCNIPortMappings(sandboxMeta.Config.GetPortMappings()),
 		})
 		if err != nil {
 			if !os.IsNotExist(err) {
 				return nil, err
 			}
-			logrus.Warnf("failed to find network namespace file %s of sandbox %s which may have been already stopped", netNSPath, podSandboxID)
+			logrus.Warnf("failed to find network namespace file %s of sandbox %s which may have been already stopped", netNSPath, sandboxContainerID)
```

Comment:
Hmmm, I found we use the level of logging without some standards in the pouch. What are the situations we use the `warn` to represent errors? What are the differences between `warn` and `Error`?

Reply:
Is there any best practice about using log level? @allencloud

Reply:
I think we should warn level here, if cannot find network namespace file, we should continue to stop the container and give a warning as a hint. 

I have read your comment  https://github.com/alibaba/pouch/issues/2149#issuecomment-430535398 :)

And this are some of my opinions:
I think the error should not happen or seldom. If it happens, the developer should be informed to deal with it;Warning is used to deal with events that have a high probability of occurring, and we've done something to deal with that, just as a notification to you  "hey, look, it's happening again".

Just a few personal points of view，welcome to have a communication.:)

Reply:
Whatever `warn` or `error` is used to stand for the exception happened in pouch. As you said:
1. `warn` stands for the exception which can be handled(not guarantee success)
2. `error` stands for the exception which cannot be handled.

But, if we repeatedly handle an exception, it always cannot be resolved. Whether this message should be notified to developers? It seems like that `error` and `warn` cannot notify developers efficiently and fastly. So there is not a good reason to use `warn` to instead of `error`.

Reply:
I almost agree with @fengzixu about log definition of warn and error. But in this case: 
```
logrus.Warnf("failed to find network namespace file %s of sandbox %s which may have been already stopped", netNSPath, podSandboxID)
```
It probably likes that it causes application oddities because of not finding network namespace, but maybe automatically recovery which means already stopped. So I prefer to using warn here.